apps/web-giddh/src/app/shared/mobile-number-input/countries-data.ts

```diff
@@ -0,0 +1,2907 @@
+    /** Interface for country data */
+export interface Country {
+    /** Country name in English */
+        name: string;
+    /** ISO 3166-1 alpha-2 country code */
+        code: string;
+    /** International dialing code */
+        dialCode: string;
+    /** Unicode flag emoji */
+        flag: string;
+    /** Path to flag image */
+    flagImage?: string;
+    /** Country name in native language */
+        nativeName: string;
+    /** Regex pattern for mobile number validation */
+        pattern: RegExp;
+    /** Placeholder text for input field */
+        placeholder: string;
+    /** Minimum number length */
+        minLength: number;
+    /** Maximum number length */
+        maxLength: number;
+    }
+    /** Comprehensive list of all countries with their telephone data */
+export const COUNTRIES_DATA: Country[] = [
+    {
+        name: 'United States',
+        code: 'US',
+        dialCode: '+1',
+        flag: '🇺🇸',
+        flagImage: 'assets/images/flags/us.png',
```

Comment:
Add missing Kosovo flag asset
assets/images/flags/xk.png is referenced in apps/web-giddh/src/app/shared/mobile-number-input/countries-data.ts but not found at apps/web-giddh/src/assets/images/flags/xk.png.